projects/05-fleet-management-api/docs/stack-python.md

```diff
@@ -0,0 +1,98 @@
+# Fleet Management Software API en Python
+
+## Stack de tecnologías
+
+La ecosistema de Python ofrece muchas opciones de frameworks para hacer un API.
```

Comment:
'La ecosistema' sounds good but is not conventional, 
```suggestion
El ecosistema de Python ofrece muchas opciones de frameworks para hacer un API.
```
or Maybe 'Python ofrece muchas opciones de frameworks para hacer un API.'